mlir/include/mlir/Dialect/AMDGPU/IR/AMDGPU.td

```diff
@@ -1230,14 +1230,13 @@ def AMDGPU_ScaledMFMAOp :
 def AMDGPU_MakeDmaBaseOp :
     AMDGPU_Op<"make_dma_base", [Pure, AttrSizedOperandSegments]>,
     Arguments<(ins
-                   Arg<AnyMemRef, "buffer to read from">:$src,
-                   Variadic<Index>:$src_indices,
-                   Arg<AnyMemRef, "buffer to write to">:$dst,
-                   Variadic<Index>:$dst_indices)>,
+                   Arg<AnyMemRef, "buffer to read from">:$lds,
```

Comment:
You've got `read` and `write` still ... and I'd argue that these are in the wrong order